packages/firebase-frameworks/package.json

```
@@ -55,7 +55,7 @@
   "peerDependencies": {
     "firebase": "^9.9.0 || ^10.0.0",
     "firebase-admin": "^11.0.1 || ^12.0.0",
-    "sharp": "^0.32.1"
+    "sharp": "^0.33.4"
```

just realized ^0.33.4 would break the build for people using old CLI versions after seeing the E2E test failure in the CLI PR(https://github.com/firebase/firebase-tools/pull/7517#pullrequestreview-2219342955), to keep it backwards compatible we need to support both versions
```suggestion
    "sharp": "^0.32 || ^0.33"
```